tests/integration/models/applicationInvite.spec.ts

```diff
@@ -0,0 +1,132 @@
+// tslint:disable-next-line:import-blacklist
+import * as m from 'mochainon';
+import {
+	balena,
+	BalenaSdk,
+	givenAnApplication,
+	givenLoggedInUser,
+} from '../setup';
+const { expect } = m.chai;
+
+const TEST_EMAIL = 'user.test@example.org';
+const TEST_MESSAGE = 'Hey!, Join my app on balenaCloud';
+const TEST_ROLE = 'developer';
+
+describe('Application Invite Model', function () {
+	givenLoggedInUser(before);
+
+	describe('given no application invite', function () {
+		givenAnApplication(before);
+		describe('balena.models.application.invite.getAllByApplication()', function () {
+			it('shoud return an empty Array', function () {
+				const promise = balena.models.application.invite.getAllByApplication(1);
+				return expect(promise).to.become([]);
+			});
+
+			it('should support a callback with no options', function (done) {
+				(balena.models.application.invite.getAllByApplication as (
+					...args: any[]
+				) => any)(this.application.id, function (
+					_err: Error,
+					applicationInvite: BalenaSdk.ApplicationInvite[],
+				) {
+					expect(applicationInvite).to.deep.equal([]);
+					return done();
+				});
+			});
+		});
+
+		describe('balena.models.application.invite.create()', function () {
+			it('should create and return an application invite', async function () {
+				const response = await balena.models.application.invite.create(
+					this.application.id,
+					{
+						collaborator: TEST_EMAIL,
+						roleName: TEST_ROLE,
```

Comment:
Reminder to add:
* a test that the user isn't logged in
* a test w/ an app param for an app that the user doesn't have access to (eg: 999999)
* a test w/ an invalid role
* a test w/o a specified role